### ISSUE TITLE
error: use of undeclared identifier 'absl'

### ISSUE THREAD

**Issue by baijinjun** [google/libprotobuf-mutator#246]
/usr/local/include/absl  and  /usr/local/lib  both has absl , why it show the error when i ninja

**Comment by baijinjun** [comment]
![image](https://github.com/google/libprotobuf-mutator/assets/40147544/368fb1ab-dcea-417b-b143-7128223013cb)

**Comment by hawkinsw** [comment]
A heavyweight fix for me was to force the library to build it's own copy of the protobuf library by toggling the `LIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF` in the source's `CMakeLists.txt`. cc @Microsvuln

**Comment by Microsvuln** [comment]
@hawkinsw Yes, seems like the workaround works for me and the issue is solved!

Thank you!

**Comment by hawkinsw** [comment]
Great!! Glad that it helped!

**Comment by hartwork** [comment]
@baijinjun could you close this as fixed?
